package.json

```diff
@@ -90,18 +90,18 @@
     "report-latency": "./bin/report-latency"
   },
   "engines": {
-    "node": ">=4.0"
+    "node": ">=8.0"
   },
   "dependencies": {
     "assert-plus": "^1.0.0",
     "bunyan": "^1.8.12",
     "csv": "^1.1.1",
     "escape-regexp-component": "^1.0.2",
     "ewma": "^2.0.1",
-    "find-my-way": "^1.13.0",
+    "find-my-way": "^2.0.0",
```

Comment:
Hello! `find-my-way` maintainer here. `v2.0.0` had a security issue, we have already fixed it in `v2.0.1`. I would recommend to use it as default version since `v2.0.0` is deprecated and tools such as [snyk](https://snyk.io) will flag the project as vulnerable :)

Reply:
@delvedor thanks a ton for letting us know! I will bump it up to use `v2.0.1`